src/cloud/project/privatelink-service.md

```diff
@@ -0,0 +1,138 @@
+   -  Troubleshoot connection issues during enablement
+
+## Limitations
+
+-  Support for PrivateLink is available on Pro plan Production and Staging environments only. It is not available on local or integration environments, or on Starter plan projects.
+-  You cannot establish SSH connections using PrivateLink. Due to infrastructure requirements, Magento cannot open Port 22 for communication between VPCs.
+-  Magento support does not cover troubleshooting AWS PrivateLink issues beyond initial enablement. Customers have the option to purchase additional support from Magento Services.
+-  Customers are responsible for costs associated with managing their own VPC.
+
+## PrivateLink connection types
+
+The following network diagram shows the PrivateLink connection types available to establish secure communication between the Magento store and external systems hosted outside of the Cloud environment.<br><br>
+
+![PrivateLink network diagram]
+
+For Pro plan Production and Staging environments, you must specify the PrivateLink connection type required for your environments:
+
+-  **Unidirectional PrivateLink**–Choose this configuration to retrieve data securely from a Magento Commerce store.
+-  **Bidirectional PrivateLink**–Choose this configuration to establish secure connections to and from systems outside of the Magento Cloud environment. The bidirectional option requires two connections:
+   -  A connection between your VPC and the Magento VPC
+   -  A connection between the Magento VPC and your VPC
+
+{:.bs-callout-tip}
+Work with your network administrator and Magento CSM to determine the right PrivateLink connection type for your project. Also, see your Cloud platform PrivateLink documentation [AWS PrivateLink][], [Azure PrivateLink].
+
+## Request PrivateLink enablement
+
+{:.bs-callout-warning}
+Enabling PrivateLink can take up to 5 business days. Providing incomplete, or inaccurate information when you request enablement can delay the process.
+
+### Prerequisites
+
+-  A Cloud account (AWS or Azure) in the same region as the {{site.data.var.ece}} instance
+-  A VPC in the customer environment that hosts the services to connect via PrivateLink
+-  For bidirectional PrivateLink connections, your must configure the PrivateLink service in your VPC environment before requesting Magento PrivateLink enablement. See [Customer set up for bidirectional PrivateLink communication][#customer-set-up-for-bidirectional-privateLink-communication].
+-  Gather the following data required for PrivateLink enablement:
+
+   -  **Cloud account number** (AWS or Azure)–Must be in the same region as the Magento Cloud instance
+   -  **Cloud region**–Provide the Cloud region where the account is hosted for verification purposes
+   -  **Services and communication ports**–Magento must open ports to enable service communication between VPCs, for example _Webserver, HTTP port 80_, _SFTP port 2222_
+   -  **Magento Cloud Project ID**
```

Comment:
The only one without a statement. Maybe can identify where to get the ID just so there is something there.